nativelib/src/main/resources/scala-native/libunwind/Unwind-EHABI.h

```diff
@@ -16,7 +16,9 @@
 #if defined(_LIBUNWIND_ARM_EHABI)
 
 #include <stdint.h>
-#include <unwind.h>
+// #include <unwind.h>
+// Change from original, point to local header file
+#include "include-libunwind/unwind.h"
```

Comment:
```suggestion
#include "include-libunwind/unwind.h"
```
If think this comment is not really needed. It's the last place in the whole codebase where `include <unwind.h>` existed. In all other places, the local header file was always included alredy.

Reply:
Sure, that is fair. Plus, if we go to upgrade the changes will be pretty obvious.